### PR TITLE
Add period option to statistics card

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Allows to choose the period of the statistics

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts [hunk 1]
```diff
@@ -41,11 +42,21 @@ const cardConfigStruct = assign(
     entities: array(entitiesConfigStruct),
     title: optional(string()),
     days_to_show: optional(number()),
+    period: optional(
+      union([
+        literal("5minute"),
```
Comment (spacegaier) [inline src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:47]: Shouldn't this be "5minute**s**" (plural)?

Comment (bramkragten) [reply src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:47]: Then we would differ from the backend options, and I don't think we should do that. The UI will show minutes.

Comment (spacegaier) [reply src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:47]: Ah, wasn't aware that the backend defines that value.

**Comment by zsarnett** [comment]
image

**Review by zsarnett — APPROVED** [review]

**Pull request merged**